packages/lib-sourcify/src/Validation/SolidityMetadataContract.ts

```diff
@@ -261,11 +268,12 @@ export class SolidityMetadataContract {
     };
 
     if (!compilationTarget || Object.keys(compilationTarget).length != 1) {
-      throw new Error(
+      logDebug(
         `Can't create JsonInput from metadata: Invalid compilationTarget in metadata: ${Object.keys(
           this.metadata.settings.compilationTarget,
         ).join(',')}`,
       );
```

Comment:
it's better to always log using the object to log logger's metadata. In this case, something like this:

```ts
logDebug(
        `Can't create JsonInput from metadata: Invalid compilationTarget in metadata`,
        {
              compilationTargets: Object.keys(
                 this.metadata.settings.compilationTarget,
              )
        }
);
```

Reply:
(It wasn't like this before because it was passed directly to the user.